new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorRestartWhenPodRoll.java

```diff
@@ -235,18 +234,14 @@ private void restartOperatorAndVerify() {
             opNamespace,
             condition.getElapsedTimeInMS(),
             condition.getRemainingTimeInMS()))
-        .until(assertDoesNotThrow(() -> operatorIsReady(opNamespace),
-          "operatorIsReady failed with ApiException"));
+        .until(assertDoesNotThrow(() -> isOperatorPodRestarted(opNamespace, opPodCreationTime),
```

Comment:
you already have operator pod name in this method, you can directly call isPodRestarted(podName, namespace, timestamp)?

Reply:
I only have the old name, and a restarted pod has a different name.

Reply:
isOperatorPodRestarted() does not require the operator pod name because we'll only have one operator in a namespace.